### DISCUSSION TITLE
Unable to edit or add new meeting

### DISCUSSION THREAD

**Discussion by SusiePBCaa** [code4recovery/12-step-meeting-list#1257] [Q&A] [answered]
Hello, I am seeking help as to why I cannot edit or add a meeting. I searched this site and saw a similar issue, but not quite the same one. I checked to see if their solution would work for me and it did not. 

I edited a virtual meeting to include in-person, and after clicking “update,” the address field went blank, and my change would not take. So I went to “add post” to add a new meeting, and after publishing it, I see it on the list behind the scenes, but it does not show up on the meeting list that people see. When I click “edit” or “view” on the post, I get the message, “meeting not found.” 

Our website is aa-palmbeachcounty.org. It is on Wordpress Version 6.4.1, the User Interface Display is set to TSML UI, and the version of the 12 Step Meeting List plug-in is 3.14.25. 

I thought about restoring the site to the November 7th backup, since we were able to add a meeting on that day without a problem, but since I don’t know much about this sort of thing, I decided to ask here first. Any help you can give is greatly appreciated. 

Thank you, 
Susie
![Meetings behind the scenes screenshot](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/d7cf60cb-c264-4ae4-9ae3-7a0dbd69887b)
![Meetings public list screenshot](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/7a5ab7ba-778b-40d0-ac6f-fc73f0f44644)

**Comment by mattbylett** [comment]
Hi Susie, did you manage to resolve this issue? seems like they are showing now! 
https://aa-palmbeachcounty.org/meetings/?search=here+and+now

**Reply by SusiePBCaa** [reply]
Thank you, I do see that it is showing now. Maybe I just need to wait a bit for it to show up. I just entered 2 more meetings and am getting the same thing as before. I shall wait and see. Thank you!

**Comment by mattbylett** [comment]
Doesn't sound optimal.  Let me see if I can replicate and I'll chat with the team.  Thank you for your patience :)

**Comment by mattbylett** [comment]
Hi Susie..  ok it’s not normal behaviour for the meeting to delay in posting.  The other thing that is causing us concern is when you say about the title going blank and the update doesn’t take… please can you explain a bit more about that ?

as for the delayed visibility… could be caching and / or could be plugin conflict - here are a couple of things you can try:

1. try in a private / incognito window and see if the meetings are visible there - that will let us know if it’s cache 
2. Disable all other plugins (temporarily) to see if there is a clash.  If the meetings are now visible turn the plugins back on 1 by 1 until you find the culprit 

if you do find it’s the result of a plugin conflict please let us know what it is and we can investigate further

let me know how you get on :)
Matt

**Reply by SusiePBCaa** [reply]
Thank you for your replies. To expand on what happened, I attempted to edit a virtual meeting to include in-person. I checked "Yes" to "Can I currently attend this meeting in person," entered the church where the meeting is located, and the address and map automatically came up. Everything looked good, When I clicked "Update," the screen returned to the location and address fields, showing that they were blank. I tried again and the same thing happened. After investigating and not being able to figure it out, I deleted the meeting and tried entering a new one. The same thing kept happening with the address fields after clicking "Publish." I left the page and saw that it was listed on the plug-in's meeting list. I then clicked on "view" to see if the address showed up anyway, and got the message "meeting not found."

Also, the meetings I entered on Friday still are not showing up on the front end. I tried the incognito mode and got the same result. 
![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/eb4f3784-5939-4426-af1f-49f49b31ce1f)

Yet they appear on the backend.
![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/6a5012d7-18a9-4902-b2a4-75b7ad93e5e8)

I did not have a chance to work on this over the weekend and I will not be able to check the plug-in's as you suggested until tomorrow (Tuesday). I will write back and let you know what I find. 

Thanks again,
Susie

**Reply by SusiePBCaa** [reply]
Well I find this to be very strange!

I disabled all plug-ins and it made no difference until I clicked "Edit" on each meeting that was not showing up, and then clicked "Update." After that, the meetings showed up when I clicked "View" and they appeared on the meeting list on our website. 

I then activated one plug-in (Jetpack Boost) and entered a new meeting called "testing." I clicked "Publish" and nothing happened. I kept trying and it continued to do nothing, remaining a "Draft." I trashed that post, deactivated Jetpack Boost, and entered a new meeting again. This time, the same thing happened that happened on Friday - it seemed to post but the address fields and map went blank, and when I click "View" on the back-end, it says "Meeting not found."  I took screenshots to show exactly what I am talking about. 

Entering a new meeting, still shows it is a draft, and the address and map are there (they show up automatically when I enter the church name):

![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/2408608f-3567-40ea-97ea-ce0f2dd73f91)

![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/8cc1199a-757e-4426-a8b6-aa88d2882b56)

After clicking "Publish" it shows it was published, yet the location and map go away, which did not used to happen:

![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/814e680c-6ebe-47b9-b66d-7d0a7acdf673)

![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/c881474d-4d1f-4d9b-9c97-e4f7fdce71e0)

I click on "Meetings" to see my list of meetings, and it is there:

![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/6d1dad49-471b-4ad3-b5d0-f564efa5090f)

And when I click on "View" I see "Meeting not found," followed by a list of meetings coming up:

![image](https://github.com/code4recovery/12-step-meeting-list/assets/150461227/99443a10-851f-4d4f-9a45-7e77cf842a44)

I also triple-checked this, and made sure all plug-ins are disabled. 

Thank you so much for your help!

Susie

**Reply by joshreisner** [reply]
thanks susie! was this working at one point and then it stopped? or has the plugin never worked properly for palm beach county aa? 

sounds like you suspected jetpack boost was a problem but then youre still experiencing the problem even with it deactivated?

**Reply by SusiePBCaa** [reply]
It was working fine until last Thursday, November 9. We created a new meeting on November 7 and everything worked. We created a new meeting on November 9 and the problems I described above happened. If I can't figure it out by tomorrow, I am going to restore the site to November 7 and see if that works.

**Reply by joshreisner** [reply]
Hm, sorry to hear it! What options do you have enabled in Jetpack Boost? i installed it on our demo site and was able to create a meeting without a problem, but the only option i have enabled is "optimize critical css"

**Reply by joshreisner** [reply]
Hm, I enabled all the other options as well and it still seems to be working ok on our demo site.

Hearing that the location goes blank makes me think a problem with geocoding, but we have a log and i'm not seeing any geocode failures on your site.

Would you be open to creating a user account for us? if you could make `wordpress@code4recovery.org` an editor (temporarily) and email us the password, we might be able to discover something by trying the flow ourselves?

**Reply by nataliefaith2** [reply]
Hi @joshreisner I'm also having the same issue when editing or adding new meetings.

[https://al-anon.org.au/meetings/?meeting=test-meeting&v=3a1ed7090bfa
](https://al-anon.org.au/meetings/?meeting=test-meeting&v=3a1ed7090bfa
)
<img width="1195" alt="Screen Shot 2023-11-16 at 10 03 50 pm" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/5abbe28e-78c8-4dd6-b09e-ed486df0579d">

**Comment by SusiePBCaa** [comment]
I added you as a user. I believe you create the password. Something to look at if you could, is the Jetpack Boost. When I regenerate, it says a number of files could not be regenerated because they are being redirected, and these files are all meetings that were entered by someone else at the Intergroup office after I created the website. I have been trying to figure out why they are redirecting. I figured it was because of the way I did the site, and I feel like it is a simple fix, I just can't find it. I created this website using aapalmbeachcounty.org and imported our meeting list from the old website and then transferred our domain, aa-palmbeachcounty.org to this new site. Everything worked fine though until November 9. 

Thank you for checking it out!

Susie
 



On Wed, 15 Nov 2023 09:39:14 -0800, Josh Reisner ***@***.***> wrote:
 
 

Hm, I enabled all the other options as well and it still seems to be working ok on our demo site.

Hearing that the location goes blank makes me think a problem with geocoding, but we have a log and i'm not seeing any geocode failures on your site.

Would you be open to creating a user account for us? if you could make ***@***.*** an editor (temporarily) and email us the password, we might be able to discover something by trying the flow ourselves?

**Comment by SIA-WEBSUP** [comment]
@SusiePBCaa,

Have you added any new plugins recently?
Have you updated any plugins recently?
Do you know what the plugin named 'WordPress.com Editing Toolkit' does for you?
Would you be open to deactivating it and seeing if that solves the problem you are experiencing?

**Reply by nataliefaith2** [reply]
@SIA-WEBSUP As I'm experiencing the same issue, I can confirm all plugins, themes and wordpress versions are up to date. I thought this issue may have been triggered by a recent WordPress update as I noticed the "Add New Meeting" option under Meetings has changed to "Add A New Post".

@joshreisner kindly fixed [the geolocating issue](https://github.com/code4recovery/12-step-meeting-list/discussions/1263) I was experincing, but when I click "publish" and view the new/updated meeting it comes up with an error message saying "meeting cannot be found", and when i view my meetings it's changed to draft (not published) and I'm unable to publish.

**Reply by SIA-WEBSUP** [reply]
@nataliefaith2 , is your site hosted by wordpress.com? 
I believe not, but I just wanted confirmation.

**Reply by joshreisner** [reply]
> when I click "publish" and view the new/updated meeting it comes up with an error message saying "meeting cannot be found", and when i view my meetings it's changed to draft (not published) and I'm unable to publish 

this sounds like slightly different behavior than what palm beach is seeing. the meeting appears to be published, but the location is no longer linked.

**Reply by joshreisner** [reply]
do you think you could record a loom or other screencast video of what you're seeing, or invite `wordpress@code4recovery.org` to be an editor of your site temporarily?

**Reply by SIA-WEBSUP** [reply]
@joshreisner, on https://aa-palmbeachcounty.org/, what I'm seeing is that the post does actually publish.... it's listed as one of the meetings. but you can't actually find it or visit its details page.

**Reply by joshreisner** [reply]
@SIA-WEBSUP yes exactly

**Reply by joshreisner** [reply]
@SusiePBCaa we've now tested with and without the full site editing plugin and don't believe this is the problem. also jetpack and jetpack boost seem to be ok

we're kind of at a loss as to what the issue is - best we can tell it seems that when the meeting is saved it somehow is doing everything right except for the key step of associating the location with the meeting, not sure why that is without accessing the backend somehow.

we'd like to investigate wordpress.com more, but it would cost us $40 to set up one month of service to test

one somewhat related thing maybe to try: try adding wordpress@code4recovery.org on your Meetings > Settings under Change Notification Emails - this might give some intel on what it thinks it's doing when it is saving a change?

<img width="479" alt="Screenshot 2023-11-17 at 6 43 04 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/2b7684e4-3fbe-4649-9231-11454d86234d">

**Reply by joshreisner** [reply]
hi @SusiePBCaa i did just think of something more concrete actually: could you install this plugin? https://wordpress.org/plugins/query-monitor/ this will provide us insights on what is happening during the save process. it will only be visible to logged-in users on the dashboard. thanks!

**Reply by SusiePBCaa** [reply]
Hi @joshreisner, I installed the plug-in. Thanks!

**Reply by joshreisner** [reply]
thank you @SusiePBCaa ! i can't seem to see the query monitor panel, do you mind temporarily upgrading my user account to administrator level? i think probably editors are not allowed to see the queries. thanks

**Comment by SusiePBCaa** [comment]
Okay, thank you for your efforts. I will try your suggestion and let you know how it goes.
 



On Fri, 17 Nov 2023 06:45:02 -0800, Josh Reisner ***@***.***> wrote:
 
 

@SusiePBCaa we've now tested with and without the full site editing plugin and don't believe this is the problem. also jetpack and jetpack boost seem to be ok

we're kind of at a loss as to what the issue is - best we can tell it seems that when the meeting is saved it somehow is doing everything right except for the key step of associating the location with the meeting, not sure why that is without accessing the backend somehow.

we'd like to investigate wordpress.com more, but it would cost us $40 to set up one month of service to test

one somewhat related thing maybe to try: try adding ***@***.*** on your Meetings > Settings under Change Notification Emails - this might give some intel on what it thinks it's doing when it is saving a change?

**Comment by joshreisner** [comment]
hi @SusiePBCaa don't mind my earlier comment about upgrading us to Administrator, we signed up for our own account on WordPress.com and are researching this issue there. it appears to be limited only to sites running on wordpress.com. we believe there may be some kind of dynamic database caching layer that wordpress.com added recently, which is preventing our plugin from seeing the connection between new meetings and their locations. we'll keep investigating and will update you when we know more.

**Comment by SusiePBCaa** [comment]
Thank you @joshreisner 
 



On Mon, 04 Dec 2023 08:22:09 -0800, Josh Reisner ***@***.***> wrote:
 
 

hi @SusiePBCaa don't mind my earlier comment about upgrading us to Administrator, we signed up for our own account on WordPress.com and are researching this issue there. it appears to be limited only to sites running on wordpress.com. we believe there may be some kind of dynamic database caching layer that wordpress.com added recently, which is preventing our plugin from seeing the connection between new meetings and their locations. we'll keep investigating and will update you when we know more.

**Comment by joshreisner** [comment, marked as answer]
hi @SusiePBCaa ok we have a change that we think will fix the issues on WordPress.com. if you're so inclined, you can[ read about it here](https://github.com/code4recovery/12-step-meeting-list/pull/1277), but the tl;dr is that since it's a big refactor we need to test it thoroughly before we roll it out to all users.

if you would like to help us test we would welcome that. you could download this ZIP file:  
[12-step-meeting-list.zip](https://github.com/code4recovery/12-step-meeting-list/files/13625711/12-step-meeting-list.zip) and then
on your site go to Plugins > Add New Plugin > Upload Plugin > Replace existing plugin

if you do that please let us know how it goes, but feel free not to and the update should roll out in a few days as a regular plugin update
